web/src/components/RunActionsMenu/RunActionsMenu.tsx

```diff
@@ -62,6 +64,7 @@ const RunActionsMenu = ({resultId, testId, testSuiteId, testSuiteRunId, isRunVie
                 navigate(`/test/${testId}/run/${resultId}`);
               }}
               key="edit"
+              disabled={!getIsAllowed(Operation.Edit)}
```

Comment:
Minor but I think for this situations we could do `canEdit` at the top and reuse the flag for the rest